### PR TITLE
Font sizes

### LINKED ISSUE #25: #333 or #000 — make up (y)our mind!
A link inside of a header is `#000` (set on `a`), and a header is `#333` (set on `h*`).

wth.

### PR DESCRIPTION
**Pull request by gavinpatkinson**
This PR proposes the following changes:
- Update the typescale to match figma mocks
- remove default vertical margins on type
- add `user-select: none` on buttons
- update readme
- fix typos in HTML styleguide

Resolves #25, #36

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -577,7 +577,7 @@ <h2 class="pt-12 mb-12" id="neutral">Neutral colors</h2>
             <div class="col-md-2  bg-gray-60 white flex mr-2  items-center justify-center">
               <div class="text-md text-600">
                 <div class="col-md-12 center">Gray-60</div>
-                <div class="col-md-12 text-mono">#4330FC</div>
+                <div class="col-md-12 text-mono">#666666</div>
```
Comment (c-johnson) [inline index.html]: can we make this like 667 or something.

File: index.html [hunk 2]
```diff
@@ -72,17 +72,17 @@ <h2 class="pt-10">Type scale</h2>
...
+              <code>&#x3C;h3&#x3E; (16/20 px)</code>
               <h3 class="mt-2 mb-8">Heading 3</h3>
-              <code>&#x3C;h4&#x3E; (20/28 px)</code>
+              <code>&#x3C;h4&#x3E; (14/16 px)</code>
               <h4 class="mt-2 mb-8">Heading 4</h4>
-              <code>&#x3C;h5&#x3E; (16/24 px)</code>
+              <code>&#x3C;h5&#x3E; (14/16 px)</code>
               <h5 class="mt-2 mb-8">Heading 5 Medium</h5>
-              <code>&#x3C;p&#x3E; (16/24 px)</code>
+              <code>&#x3C;p&#x3E; (14/16 px)</code>
```
Comment (c-johnson) [inline index.html]: The body font line height in the actual CSS below is 18px. Is it 16px or 18px?

File: index.html [hunk 3]
```diff
@@ -72,17 +72,17 @@ <h2 class="pt-10">Type scale</h2>
...
-              <code>&#x3C;h1&#x3E; (48/64 px)</code>
+              <code>&#x3C;h1&#x3E; (24/32 px)</code>
               <h1 class="mt-2 mb-8">Heading 1</h1>
-              <code>&#x3C;h2&#x3E; (36/52 px)</code>
+              <code>&#x3C;h2&#x3E; (20/24 px)</code>
               <h2 class="mt-2 mb-8">Heading 2</h2>
-              <code>&#x3C;h3&#x3E; (24/32 px)</code>
+              <code>&#x3C;h3&#x3E; (16/20 px)</code>
               <h3 class="mt-2 mb-8">Heading 3</h3>
-              <code>&#x3C;h4&#x3E; (20/28 px)</code>
```
Comment (c-johnson) [inline index.html]: H4, H5, and Body seem redundant (they're the same size/line-height). 

Maybe remove boldness changes from general type scale?

File: src/_typography.scss [hunk 4]
```diff
@@ -135,47 +135,47 @@ h5,
...
 h1,
 .h1 {
-  font-size: 12rem; //48
+  font-size: 6rem;
   font-weight: 600;
-  line-height: 16rem; //64
-  margin: 4rem 0;
+  line-height: 8rem;
+  margin: 0;
 }
```
Comment (c-johnson) [inline src/_typography.scss]: Add a ".markdown" class that wraps dynamic markdown content, which adds these heading spacing rules back in. (Most of the time we just don't need these margins)

File: index.html (hunk 3 above)
Comment (gavinpatkinson) [reply index.html]: We have removed H5

File: src/_typography.scss [hunk 5]
```diff
@@ -135,47 +135,47 @@ h5,
 .text-heading {
   margin: 0;
   font-family: "Work Sans";
-  color: $gray;
+  // color: $gray;
```
[the remark this answers is not in the document]
Comment (gavinpatkinson) [reply src/_typography.scss]: changed to black

**Comment by gavinpatkinson** [comment]
@c-johnson your comments have been resolved in `9816d29`

**Pull request merged**